tests/integration/workflow/evaluator_optimizer/test_evaluator_optimizer.py

```diff
@@ -233,6 +233,73 @@ async def agent_function():
 
     await agent_function()
 
+@pytest.mark.integration
+@pytest.mark.asyncio
+async def test_check_markup_config(fast_agent):
```

Comment:
just a copy of the test above with [this](https://github.com/evalstate/fast-agent/pull/155/files#r2081810666) addition.